src/Tabs/RoomTab.as

```diff
@@ -202,6 +203,19 @@ class RoomTab : Tab {
         @scriptSettings = room['scriptSettings'];
     }
 
+    void OnClickExportList() {
+        auto json = Json::Object();
+        json["maps"] = Json::Array();
+        for (uint i = 0; i < lazyMaps.Length; i++) {
+            json["maps"].Add(lazyMaps[i].ToJson());
+        }
+        auto folderLocation = IO::FromDataFolder("BRM_Map_Export");
```

Comment:
For this sort of thing, you should almost always use FromPluginStorageFolder. 

FromDataFolder is (IIRC), the game install directory (with GameData and Trackmania.exe). We don't want to put plugin generated exports there.